ProjectSetup/Controllers/V1/PostController.cs

```diff
@@ -79,20 +79,24 @@ public async Task<IActionResult> DeletePostById([FromRoute] Guid postId)
 		[HttpPut(ApiRoutes.Post.Update)]
 		[CustomExceptionFilter(typeof(PostBadRequestException), HttpStatusCode.BadRequest)]
 		[CustomExceptionFilter(typeof(CategoryBadRequestException), HttpStatusCode.BadRequest)]
-		public async Task<IActionResult> Update([FromBody] UpdatePostRequest postRequest)
+		public async Task<IActionResult> Update([FromBody] UpdatePostRequest updateRequest)
 		{
-			var userOwnsPost = await _repository.Post.UserOwnsPostAsync(postRequest.Id, HttpContext.GetUserId());
+			var userOwnsPost = await _repository.Post.UserOwnsPostAsync(updateRequest.Id, HttpContext.GetUserId());
 
 			if (!userOwnsPost)
 			{
 				throw new PostBadRequestException("You don't own this post");
 			}
 
-			var post = await _repository.Post.UpdatePost(postRequest);
+			var post = await _repository.Post.FindPostByIdAsync(updateRequest.Id);
```

Comment:
Do you check somewhere if post == null?

Reply:
Yes, that check is in the method FindPostByIdAsync().

Reply:
And will it throw an exception or will continue to the mapper in next line trying to map null?

Reply:
Yes, it will throw PostNotFoundException